src/model/UInt64.ts

```diff
@@ -84,6 +84,27 @@ export class UInt64 {
         return uint64.toHex(this.toDTO());
     }
 
+    /**
+     * Get string representation of number (by default, decimal representation)
+     *
+     * @return {string}
+     */
+    public toString(radix: number = 10): string {
+        var result = '';
+        var high = this.higher;
+        var low = this.lower;
+        while (true) {
```

Comment:
while (true)?? Don't see it as good practice

Reply:
I could change it to a `do while` loop: we need at least 1 iteration, so the break condition is at the end.